Header header
uint32 node_id
